### ISSUE TITLE
error 500 url is not defined

### ISSUE THREAD

**Issue by LazyGeniusMan** [nuxt-themes/alpine#100]
This is happen after `v1.1.3`. My `v1.1.0` deployment is fine.

the playground also have this problem
![image](https://user-images.githubusercontent.com/58850480/214726474-6083069c-5b99-4b8f-8ac2-06a2071cb549.png)

pnpm dev log:
```
[Vue warn]: Unhandled error during execution of watcher callback                    07:18:36
  at <AppLayout >
[Vue warn]: Unhandled error during execution of setup function                      07:18:36
  at <AppLayout >
[nitro] [dev] [unhandledRejection] ReferenceError: url is not defined               07:18:36  
    at refreshHead (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@nuxt+content@2.4.0\node_modules\@nuxt\content\dist\runtime\composables\head.mjs:90:67)
    at callWithErrorHandling (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:157:22)
    at callWithAsyncErrorHandling (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:166:21)
    at doWatch (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:1752:13)
    at Module.watch (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:1663:12)
    at Module.useContentHead (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@nuxt+content@2.4.0\node_modules\@nuxt\content\dist\runtime\composables\head.mjs:107:25)
    at setup (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@nuxt-themes+alpine@1.1.3\node_modules\@nuxt-themes\alpine\components\AppLayout.vue:33:27)
    at _sfc_main.setup (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@nuxt-themes+alpine@1.1.3\node_modules\@nuxt-themes\alpine\components\AppLayout.vue:84:23)
    at callWithErrorHandling (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:157:22)
    at setupStatefulComponent (C:\Users\LazyGeniusMan\git\website\node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:7176:29)
```

pnpm generate:
```
i Prerendering 8 initial routes with crawler                                  nitro 07:23:25

 ERROR  url is not defined                                                          07:23:26

  at refreshHead (/C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/server.mjs:23880:67)
  at callWithErrorHandling (node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.prod.js:15:22)
  at callWithAsyncErrorHandling (node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.prod.js:24:21)
  at doWatch (node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.prod.js:1178:13)
  at watch (node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.prod.js:1106:12)
  at useContentHead (/C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/server.mjs:23895:3)
  at setup (/C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/server.mjs:23913:5)
  at _sfc_main$2.setup (/C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/server.mjs:23952:25)
  at callWithErrorHandling (node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.prod.js:15:22)
  at setupStatefulComponent (node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.prod.js:5762:29)
  at setupComponent (node_modules\.pnpm\@vue+runtime-core@3.2.45\node_modules\@vue\runtime-core\dist\runtime-core.cjs.prod.js:5743:11)
  at renderComponentVNode (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:169:17)
  at ssrRenderComponent (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:621:12)
  at _sfc_ssrRender (/C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/server.mjs:24084:9)
  at renderComponentSubTree (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:251:17)
  at renderComponentVNode (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:185:16)
  at ssrRenderComponent (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:621:12)
  at default (/C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/server.mjs:24131:19)
  at ssrRenderSuspense (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:718:9)
  at /C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/server.mjs:24124:7
  at renderComponentSubTree (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:251:17)
  at renderComponentVNode (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:185:16)
  at renderToString (node_modules\.pnpm\@vue+server-renderer@3.2.45_vue@3.2.45\node_modules\@vue\server-renderer\dist\server-renderer.cjs.prod.js:447:26)
  at Object.renderToString$1 [as renderToString] (/C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/renderer.mjs:345:24)
  at Object.renderToString (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/vue-bundle-renderer@1.0.0/node_modules/vue-bundle-renderer/dist/runtime.mjs:173:40)
  at async /C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/renderer.mjs:416:21
  at async /C:/Users/LazyGeniusMan/git/website/.nuxt/prerender/chunks/renderer.mjs:59:22
  at async Object.handler (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/h3@1.0.2/node_modules/h3/dist/index.mjs:723:19)
  at async toNodeHandle (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/h3@1.0.2/node_modules/h3/dist/index.mjs:798:7)
  at async ufetch (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/unenv@1.0.1/node_modules/unenv/runtime/fetch/index.mjs:9:17)
  at async generateRoute (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/nitropack@2.0.0/node_modules/nitropack/dist/shared/nitro.5d7549ce.mjs:3892:17)
  at async prerender (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/nitropack@2.0.0/node_modules/nitropack/dist/shared/nitro.5d7549ce.mjs:3947:22)
  at async /C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/nuxt@3.1.1/node_modules/nuxt/dist/index.mjs:1763:7
  at async build (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/nuxt@3.1.1/node_modules/nuxt/dist/index.mjs:2467:5)
  at async Object.invoke (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/nuxi@3.1.1/node_modules/nuxi/dist/chunks/build.mjs:56:5)
  at async Object.invoke (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/nuxi@3.1.1/node_modules/nuxi/dist/chunks/generate.mjs:32:5)
  at async _main (/C:/Users/LazyGeniusMan/git/website/node_modules/.pnpm/nuxi@3.1.1/node_modules/nuxi/dist/cli.mjs:51:20)

```

**Comment by Atinux** [comment]
cc @farnabaz

**Comment by ZainW** [comment]
i was having this issue as well, i ended up switching from pnpm to npm to fix this temporarily

not sure if its an actual pnpm bug or not

**Comment by Atinux** [comment]
This is due to Nuxt Content v3.4.1, fixed in v3.4.2

Related PR to fix #101

**Comment by LazyGeniusMan** [comment]
with v1.2.0, prerendering initial routes is now successful but now I got this error and the build failed completely

```
05:58:15.703 | Validating asset output directory
05:58:17.124 | Failed: build output directory contains links to files that can't be accessed
```
I use cloudflare pages with this build setting:
![image](https://user-images.githubusercontent.com/58850480/215222327-66c614df-f15c-46b3-b0a4-fd6deebaeb30.png)
This is what I used in my successful deployment using alpine v1.1.0 and I have not changed anything beside alpine version
